README.md

```diff
@@ -28,7 +28,7 @@ The output of this repository is an npm package (`@educandu/educandu`).
  | cdnBucketName | The name of the AWS S3 bucket storing the CDN data | `string` | yes |
  | cdnRootUrl | The root url of the CDN | `string` | yes |
  | customResolvers | The same object that is also used to hydrate the app on the client side | `{ resolveCustomPageTemplate, resolveCustomHomePageTemplate, resolveCustomSiteLogo, resolveCustomPluginInfos }` | yes, accepts `null` for either property and it will default to the internal setup
- | publicFolders | The project-specific public folders that need to be accesible on the project domain | `array` of string | no |
+ | publicFolders | The project-specific public folders that need to be accesible on the project domain | `array` of `{ publicPath: <string>, destination: <string>, setHeaders: <function(res, path){}> }` | no, also `setHeaders` is optional |
```

Comment:
I does not go unnoticed that you have updated the README file! 💟

Reply:
Tihi :smile: